sites/cheerpx/src/content/docs/12-reference/00-CheerpX-Linux-create.md

```diff
@@ -78,4 +78,116 @@ networkInterface?: NetworkInterface;
 
 This option configures network settings, which allows CheerpX to communicate over networks. For more detailed information about how CheerpX handles networking, including the use of Tailscale and overcoming browser limitations, see the [Networking](/docs/guides/Networking) guide.
 
+### `authKey`
+
+```ts
+authKey?: string;
+```
+
+The `authKey` is an optional string used for authentication and should be passed directly in the `create` call.
+
+Example:
+
+```js
+const cx = await CheerpX.Linux.create({
+	mounts: mountPoints,
+	networkInterface: { authKey: "YOUR KEY" },
+});
+```
+
+In this example, the `authKey` is included in the `networkInterface` object to enable authentication for the CheerpX instance.
+
+### `controlUrl`
+
+```ts
+controlUrl?: string;
+```
+
+The `controlUrl` is an optional string used to specify the URL of a self-hosted server.
```

Comment:
```suggestion
The `controlUrl` is an optional string used to specify the URL of a self-hosted Headscale server.
```

Also link back to the relevant guide. The short description in the Networking guide it's also better here